commands/web/phpstan

```diff
@@ -18,4 +18,15 @@ test -e phpstan.neon || curl -OL https://git.drupalcode.org/project/gitlab_templ
 sed -i 's/BASELINE_PLACEHOLDER/phpstan-baseline.neon/g' phpstan.neon
 # Add an empty baseline file to ensure it exists.
 test -e phpstan-baseline.neon || touch phpstan-baseline.neon
-phpstan analyse $DDEV_DOCROOT/$DRUPAL_PROJECTS_PATH "$@"
+
+CONTRIB_MODULE_DIR=$(find "$DDEV_DOCROOT/$DRUPAL_PROJECTS_PATH" -mindepth 1 -maxdepth 1 -type d | head -n 1)
```

Comment:
I think we can avoid the `find` by using same approach with DDEV_SITENAME as in this line https://github.com/ddev/ddev-drupal-contrib/blob/bdcebb2fce63a164ab9765f32a71768fc0f038a3/commands/web/symlink-project#L19

Reply:
Switched to `DDEV_SITENAME` and I think it would worth opening a follow up ticket for creating a env var when containers start with the name of the extension directory (`${DDEV_SITENAME//-/_`) or even with the full path to the extension directory inside the container - if you agree.

Reply:
Agree